pkg/commands/option/db.go

```diff
@@ -9,23 +9,25 @@ import (
 
 // DBOption holds the options for trivy DB
 type DBOption struct {
-	Reset          bool
-	DownloadDBOnly bool
-	SkipDBUpdate   bool
-	Light          bool
-	NoProgress     bool
-	DBRepository   string
+	Reset           bool
+	DownloadDBOnly  bool
+	SkipDBUpdate    bool
+	InsecureTlsSkip bool
```

Comment:
We already have `Insecure` here. I think we can use it.
https://github.com/aquasecurity/trivy/blob/54c5ffa5b8e78e1d2a69b5beda826039ae1ac771/pkg/commands/option/artifact.go#L17

Reply:
[Config struct](https://github.com/aquasecurity/trivy/blob/f8951f38475d1c9fbf1e2f6d0c4c92b10b2d3712/pkg/commands/server/config.go#L10-L18) doesn't contain `ArtifactOption`. That is why i added new field to `DBOption` struct.